src/server/admin/server/convert1_9.go

```diff
@@ -203,19 +203,30 @@ func convert1_9Secrets(secrets []*pps1_9.Secret) []*pps.SecretMount {
 	return result
 }
 
+func copyStringSlice(ss []string) []string {
+	if ss == nil {
+		return nil
+	}
+	result := make([]string, 0, len(ss))
+	for _, s := range ss {
+		result = append(result, s)
+	}
```

Comment:
```suggestion
	result := make([]string, len(ss))
	copy(result, ss)
```

I think this is a little terser and in theory a little more efficient (not that it really matters here.)

Reply:
I decided to follow the same pattern found throughout this file to be consistent.